pkg/verify/verify.go

```diff
@@ -283,9 +223,54 @@ func (v *releaseVerifier) Verify(ctx context.Context, releaseDigest string) erro
 		}
 		return fmt.Errorf("unable to locate a valid signature for one or more sources")
 	}
+
+	v.cacheVerification(releaseDigest, signedWith)
```

Comment:
Why cache `signedWith`?  Seems like this is a boolean cache "yes, we verified that digest".  Possibly interesting metadata would include the signatures' and signing keys' expiration dates, so we knew when to invalidate the cache.

Reply:
because we're going to start saving these in a config map on cluster in the future so if the signing servers go down we can still upgrade to a version we saw in available versions.

Reply:
> because we're going to start saving these in a config map on cluster in the future...

I still don't see why that means "cache the sigs in memory".  Can we leave this off now and grow it back when we add the cache-to-ConfigMap functionality?